gui/src/installer/step/mod.rs

```diff
@@ -52,11 +52,19 @@ pub trait Step {
 }
 
 #[derive(Default)]
-pub struct Welcome {}
+pub struct Welcome {
+    path: Option<PathBuf>,
```

Comment:
This might affect the logic, but perhaps there's no need for `Option` here as it should always have a value.

Also, it might also be worth renaming the field to be consistent with other steps and/or the context object, e.g. `data_dir`?

Reply:
We need an option, as on a fresh install (or no other wallet created) there is no meaning to come back to the `Launcher`, as we directly launching the installer...